_button.scss

```diff
@@ -0,0 +1,71 @@
+// button
+// ===
+
+// Configurable Variables
+// ---
+
+//Button Styles
+//---
+
+$button__padding: $vertical-input-padding $horizontal-input-padding !default;
+
+$button__border: if(variable-exists(border), $border, 1px solid $neutral-50) !default;
+$button__border-radius: if(variable-exists(border-radius), $border-radius, 0) !default;
+
+
+$button__font-color: inherit !default;
+$button__font-family: inherit !default;
+$button__font-size: inherit !default;
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'level/level';
+
+
+// Button root
+// ---
+
+.c-button {
+    min-height: $tap-size;
+    display:inline-block;
+    margin: 0;
+    cursor:pointer;
+    background: none;
+
+    text-align:center;
+    border:none;
+    line-height: normal;
+    white-space: normal;
+
+    span {
+        font-size: $button__font-size;
+        font-family: $button__font-family;
+
+        border: $button__border;
+        border-radius: $button__border-radius;
+        padding: $button__padding;
+
+        display:inline-block;
```

Comment:
Once everything is sorted (not right now), you’ll want to check [CST’s preferred property sort order](https://github.com/mobify/mobify-code-style/blob/37a9e8a11b875eba8d43d239aff4b6aeed44f933/css/.scss-lint.yml#L290).